advanced/how_to_debug.rst

```diff
@@ -0,0 +1,144 @@
+.. _how_to_debug:
+
+How to debug
+============
+
+Console and debugger
+--------------------
+
+When games made with Escoria need testing, some errors and warnings may appear
+in the console. This information can be very useful when debugging issues.
+
+If Escoria encounters an error or a crash, it will stop the execution of the
+game and return to the editor, showing you the line that broke. Usually, this
+line will be located in Escoria's logger class, as this class is in charge of
+showing the errors before stopping execution.
+
+Godot also shows the debugger panel but it is advised to check the output log
+first.
+
+.. image:: img/crash-error-console.png
+    :align: center
+    :alt: Crash error: console
+
+The console shows many lines, the verbosity of which depends on the log level
+defined in **Project Settings**. In order from finest- to highest-level:
```

Comment:
Change the last part to from "most to least granular"

Reply:
Done!